Framework/DataHandling/src/LoadSampleEnvironment.cpp

```diff
@@ -246,12 +246,14 @@ void LoadSampleEnvironment::exec() {
   sample.setEnvironment(std::move(environment));
 
   auto translatedVertices = environmentMesh->getVertices();
-  int i = 0;
-  for (double vertex : translatedVertices) {
-    i++;
-    g_log.information(std::to_string(vertex));
-    if (i % 3 == 0) {
-      g_log.information("\n");
+  if (g_log.is(7)) {
```

Comment:
Good that we check this now. Can we use the named variant instead? `g_log.is(Logger::Priority::Debug)` is a bit clearer than `7`?

Reply:
ok,  will do

Reply:
Thanks